site2/website/versioned_docs/version-2.8.1/client-libraries-java.md

```diff
@@ -122,6 +122,9 @@ int|`connectionTimeoutMs`|Duration of waiting for a connection to a broker to be
 int|`requestTimeoutMs`|Maximum duration for completing a request |60000
 int|`defaultBackoffIntervalNanos`| Default duration for a backoff interval | TimeUnit.MILLISECONDS.toNanos(100);
 long|`maxBackoffIntervalNanos`|Maximum duration for a backoff interval|TimeUnit.SECONDS.toNanos(30)
+SocketAddress|`socks5ProxyAddress`|Socks5 proxy address | None
```

Comment:
Do we need to add descriptions as below?

If you want to use the SOCKS5 proxy, you can add a SOCKS5 proxy handler in the connection pool as the first handler for both producer and consumer.

Reply:
If user config socks5ProxyAddress, then SOCKS5 proxy handler will be added as the first handler automatically.